apisix/cli/ops.lua

```diff
@@ -916,10 +916,15 @@ local function stop(env)
 end
 
 
+function sleep(n)
+  os.execute("sleep " .. tonumber(n))
+end
+
 local function restart(env)
   -- test configuration
   test(env)
   stop(env)
+  sleep(1)
```

Comment:
use ngx.sleep() instead?

Reply:
i have try to instead  ngx.sleep  , but i got a error 

/usr/local/openresty//luajit/bin/luajit: /usr/local/apisix/apisix/cli/ops.lua:930: attempt to index global 'ngx' (a nil value)
stack traceback:
	/usr/local/apisix/apisix/cli/ops.lua:930: in function </usr/local/apisix/apisix/cli/ops.lua:925>
	/usr/local/apisix/apisix/cli/ops.lua:980: in function 'execute'
	/usr/local/apisix/apisix/cli/apisix.lua:40: in main chunk
	[C]: at 0x00405290

so what should i do ?

Reply:
never mind, it seems ngx module isn't available during this phase.